runtime/lib_ksils12.c

```diff
@@ -744,6 +830,9 @@ rsksiCtxNew(void) {
 	ctx->syncMode = LOGSIG_SYNCHRONOUS;
 	ctx->thread_started = false;
 	ctx->disabled = false;
+	ctx->ksi = NULL;
+	ctx->ksiCapacity = 0;
+	ctx->ksiCount = 0;
```

Comment:
redundant, as calloc is used.

Reply:
`ksiCapacity` and `ksiCount` removed but `ksi` is still set to `NULL`. This guarantees that code will work even if calloc is replaced with malloc.